_includes/footer.html

```diff
@@ -65,7 +65,7 @@
     <p class="text-center">
       <strong>
         <a
-          href="mailto:hello@djangocon.us">hello@djangocon.us</a>
+          href="mailto:{{site.contact_us_email}}">hello@djangocon.us</a>
```

Comment:
Ditto, suggestion about matching the 2nd hello@ reference to avoid conflicts if we change the address.